### PR TITLE
Add checkbox icon to timeout control 

### PR DESCRIPTION
**Pull request by ycleptkellan**
We recently changed the `CheckboxElement` component to use icons to represent a checked box. This PR changes the `TimeoutControl` component to do the same 👍

### CODE REVIEW

**Comment by ycleptkellan** [comment]
Got it, will do 👍

**Comment by ycleptkellan** [comment]
@ericnewcomer this bloke 's smashing to go

File: src/component/form/CheckboxElement.tsx [hunk 1]
```diff
@@ -1,58 +1,78 @@
...
 
         this.onChange = this.onChange.bind(this);
     }
 
-    private onChange(event: any): void {
-        this.setState(
-            { checked: !this.state.checked },
-            () => this.props.onChange && this.props.onChange(this.state.checked)
-        );
+    private onChange(): void {
```
Comment (ericnewcomer) [inline src/component/form/CheckboxElement.tsx]: `handleClick`

**Comment by ycleptkellan** [comment]
Ah, right 🤦‍♂️

**Pull request merged**